src/Microsoft.Bcl.HashCode/README.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft.Bcl.HashCode
```

Comment:
Hmm. That's not the package readme template that we standardized on in runtime and other repos. See https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/src/PACKAGE.md

Reply:
Oh good point. I incorrectly checked the readme in the root folder of the assemblies.

Reply:
Will address separately.